packages/space-opera/karma.conf.js

```diff
@@ -22,22 +22,32 @@
 module.exports = function(config) {
   config.set({
     basePath: '',
+    reporters: ['spec'],
+    specReporter: {
+      maxLogLines: 10,              // limit number of lines logged per test
+      suppressErrorSummary: false,  // do not print error summary
```

Comment:
Are these comments correct? It seems like setting suppress to false would print an error summary.

Reply:
whoops, lemme clean that up